tests/spread/extensions/snaps/glib-hello/snap/snapcraft.yaml

```diff
@@ -5,7 +5,6 @@ description: This is a basic gnome snap (*only* uses glib). It simply prints a h
 
 grade: devel
 confinement: strict
-base: core18
```

Comment:
The gnome extension works on core i assume?

Reply:
no, the logic is applied on "prepare"